lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -0,0 +1,164 @@
+;(function (define) {
+'use strict';
+
+define(['backbone',
+        'underscore',
+        'gettext',
+        'js/views/fields',
+        'teams/js/models/team',
+        'text!teams/templates/edit-team.underscore'],
+       function (Backbone, _, gettext, FieldViews, TeamModel, edit_team_template) {
+           return Backbone.View.extend({
+
+               maxTeamNameLength: 255,
+               maxTeamDescriptionLength: 300,
+
+               events: {
+                   "click .action-primary": "createTeam",
+                   "click .action-cancel": "goBackToTopic"
+               },
+
+               initialize: function(options) {
+                   this.courseId = options.teamParams.courseId;
+                   this.teamsUrl = options.teamParams.teamsUrl;
+                   this.topicId = options.teamParams.topicId;
+                   this.languages = options.teamParams.languages;
+                   this.countries = options.teamParams.countries;
+
+                   _.bindAll(this, "goBackToTopic", "createTeam");
+
+                   this.teamNameField = new FieldViews.TextFieldView({
+                       model: new TeamModel(),
```

Comment:
Rather than have five unique `TeamModel` objects and posting manually with $.ajax, I suggest the following: create a team model scoped to the view, and then use the model's [save](http://backbonejs.org/#Model-save) method.